lib/node_modules/@stdlib/utils/async/parallel/README.md

```diff
@@ -0,0 +1,297 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# Parallel
+
+> Execute functions in parallel, passing the results of all functions to a final callback.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var parallel = require( '@stdlib/utils/async/parallel' );
+```
+
+#### parallel( functionArray, \[options,] done )
+
+Executes functions inside `functionArray` in parallel, passing the results of all functions to a final callback.
+
+The first argument to each function is a `resolve` callback. The `resolve` callback should be invoked upon function completion. The first argument is reserved as an `error` argument (which can be `null` or void). If there is a result, it should be provided as the second argument to the `resolve` callback. Failure to invoke the `resolve` callback will prevent the `done` callback from being invoked.
+
+```javascript
+function a( resolve ) {
+    setTimeout( onTimeout, 0 );
+    function onTimeout() {
+        resolve( null, 2 );
+    }
+}
+
+function b( resolve ) {
+    setTimeout( onTimeout, 0 );
+    function onTimeout() {
+        resolve( null, 4 );
+    }
+}
+
+function done( error, out ) {
+    if ( error ) {
+        throw error;
+    }
+    console.log( out );
+    // => [ 2, 4 ]
+}
+
+var fcns = [ a, b ];
+
+parallel( fcns, done );
+```
+
+The `resolve` callback accepts two arguments: `error` and `value`. The second argument to the `resolve` callback is the output of the function. If a function function calls the `resolve` callback with a truthy error argument, the function stops executing any queued functions in case of a limit being present and calls the `done` callback for error processing.
+
+```javascript
+function a( resolve ) {
+    setTimeout( onTimeout, 0 );
+    function onTimeout() {
+        resolve( new Error( 'beep' ) );
+    }
+}
+
+function b( resolve ) {
+    setTimeout( onTimeout, 0 );
+    function onTimeout() {
+        resolve( null, 4 );
+    }
+}
+
+function done( error, out ) {
+    if ( error ) {
+        console.error( error.message );
+    }
+    // => 'beep'
+}
+
+var fcns = [ a, b ];
+
+parallel( fcns, done );
+```
```

Comment:
This appears to be a duplicated example. Am I wrong here?

Reply:
@kgryte This example showcases the error-handling. Should it be removed?

Reply:
I think we can remove. We typically don't showcase exception behavior in README examples.